packages/webapp/components/footer/FooterNavBarTabs.tsx

```diff
@@ -56,7 +58,10 @@ export const tabs: (FooterTab | ReactNode)[] = [
     icon: (active: boolean) => <Notifications active={active} />,
   },
   {
-    path: '/squads',
+    path: ({ isLaptop }) =>
+      isLaptop
```

Comment:
I've made these changes to this other PR just slightly different since we don't use this in laptop anyways.
https://github.com/dailydotdev/apps/pull/3615